scanner/src/components/Button/index.svelte

```diff
@@ -24,6 +23,7 @@
     align-items: center;
     justify-content: center;
     gap: 0.5rem;
+    font-weight: bold;
```

Comment:
As andrew mentioned elsewhere, it's easier to get the exact right font if you use numbers for weight, e.g. `700` - which you should get from Figma.